packages/flutter_tools/lib/src/migrations/swift_package_manager_integration_migration.dart

```diff
@@ -246,9 +242,21 @@ class SwiftPackageManagerIntegrationMigration extends ProjectMigrator {
     //     ReferencedContainer = "container:Runner.xcodeproj">
     // </BuildableReference>
     final List<String> schemeLines = LineSplitter.split(schemeContent).toList();
-    final int index = schemeLines.indexWhere(
-      (String line) => line.contains('BlueprintIdentifier = "$_runnerNativeTargetIdentifier"'),
```

Comment:
Since the scheme cannot rely on the constant anymore, we have to parse the identifier.